mautic/mautic
# [UI] fix: Add recipient too wide @escopecz

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This pull request addresses an issue where buttons were expanding to full width when placed next to a field label, leading to a cluttered UI. The goal is to ensure that buttons only take up space proportional to their title






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Contacts page, create a new contact, save it, and then close it. Send an example and verify that the "Add recipient" label is the same width as the content.
